### PR TITLE
Add LTS documentation (fix #670)

### LINKED ISSUE #670: LTS strategy
As mentioned in https://github.com/fastify/fastify/issues/229#issuecomment-358108181, it would be a good idea to have a published LTS strategy. Even if it is as simple as "we only support the active X release of semantic version X.Y.Z releases."

Use this issue to hash it out.

### PR DESCRIPTION
**Pull request by jsumners**
This PR adds a document to codify the LTS strategy outlined in #670. The included table should eventually look something like this:

| Version | Release Date | End Of LTS Date |
|:--------|:-------------|:----------------|
| 1.0.0 | 2018-02-01 | 2019-02-27 |
| 2.0.0 | 2018-08-31 | TBD |

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/LTS.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Fastify's Long Term Support (LTS) is provided according the schedule laid
+out in this document:
+
+1. Major releases, "X" release of [semantic versioning][semver] X.Y.Z release
+versions, are supported for a minimum period of six months from their release
+date. The release date of any specific version can be found at
+[https://github.com/fastify/fastify/releases](https://github.com/fastify/fastify/releases).
+
+1. Major releases will receive security updates for an additional six months
```
Comment (allevo) [inline docs/LTS.md]: `2.`

**Comment by allevo** [comment]
Why not a `rc` version ?

File: docs/LTS.md (hunk 1 above)
Comment (jsumners) [reply docs/LTS.md]: It gets auto corrected by the Markdown renderer.

**Comment by jsumners** [comment]
@allevo why would we LTS an `rc` version?

**Comment by allevo** [comment]
Yeah sorry. We don't need to publish the `rc` version.

File: docs/LTS.md (hunk 1 above)
Comment (jsumners) [reply docs/LTS.md]: FYI, the reason for letting the Markdown renderer handle it is to make it easier to rearrange points in the future.

**Pull request merged**